src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -962,6 +1063,15 @@ Whether you're considering New Relic One or you're already using our capabilitie
     For in-depth scriptable testing, including real browser tests and tests of API endpoints, see [Synthetic Monitoring](/docs/synthetics/new-relic-synthetics/getting-started/new-relic-synthetics). Synthetic Monitoring includes free ping monitoring, which allows you to monitor your website from locations around the world. For more information, see [Types of Synthetic monitors](/docs/synthetics/new-relic-synthetics/getting-started/types-synthetics-monitors).
   </Collapser>
 
+  <Collapser
+    id="policy"
+    title="policy"
+  >
+   A policy is bucket for similar conditions so that you can see them all in one place. So, if your team wants to examine all of the conditions for your Java application, you will need to create a policy to contain those conditions.  
```

Comment:
```suggestion
   A policy is a bucket for similar conditions so that you can see them all in one place. So, if your team wants to examine all of the conditions for your Java application, you need to create a policy to contain those conditions.  
```